### PR TITLE
Adaptive Histogram Equalization

### PR DESCRIPTION
**Pull request by khilanravani**
It is used to improve contrast in images and works by performing enhancement on small 'contextual' regions or neighborhood of each pixel in the given image.

### CODE REVIEW

**Comment by khilanravani** [comment]
Sample input/output : 
input
ahe

File: src/Graphics/Image/Processing/Ahe.hs [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+-- The idea is to perform contrast enhancement in 'neighborhood region' of each pixel and the size
+-- of the region is a parameter of the method. It constitutes a characteristic length scale: contrast 
+-- at smaller scales is enhanced, while contrast at larger scales is reduced (For general purposes, a size
+-- factor of 5 tends to give pretty good results).
+--
+-- <<images/yield.jpg>>   <<images/yield_ahe.png>>
+--
+-- Usage : 
+--	
+-- >>> img <- readImageY VU "yield.jpg"
```
Comment (lehins) [inline src/Graphics/Image/Processing/Ahe.hs]: This should be `"images/yield.jpg"`

File: src/Graphics/Image/Processing/Ahe.hs [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+-- >>> input1 <- getLine
+-- >>> input2 <- getLine
+-- >>> let thetaSz = (P.read input1 :: Int)
+-- >>> let distSz = (P.read input2 :: Int) 
+-- >>> let neighborhoodFactor = (P.read input2 :: Int) 
+-- >>> let aheImage :: Image VU RGB Double
+-- >>>     aheImage = ahe img thetaSz distSz neighborhoodFactor
+-- >>> writeImage "test.png" (toImageRGB aheImage)
```
Comment (lehins) [inline src/Graphics/Image/Processing/Ahe.hs]: And this should be `"images/yield_ahe.png"`, since that's what it is called in the repo.

**Comment by lehins** [comment]
@khilanravani This PR looks great. It seems though that you added another copy of `yield.jpg` image to the PR by mistake, since it is already in `images/yield.jpg`.

File: src/Graphics/Image/Processing/Ahe.hs (hunk 1 above)
Comment (khilanravani) [reply src/Graphics/Image/Processing/Ahe.hs]: Yes sir, I'll change it.

File: src/Graphics/Image/Processing/Ahe.hs (hunk 2 above)
Comment (khilanravani) [reply src/Graphics/Image/Processing/Ahe.hs]: Yes sir, very sorry. Will change it.

**Comment by khilanravani** [comment]
@lehins Thank u sir! Yes sir, I'm very sorry. I'll correct it.

**Pull request merged**